### PR TITLE
Limit desktop editor options width on large viewports

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

This will limit the desktop editor options width, following the practice we've used in acount settings[[1](https://github.com/gitpod-io/gitpod/blob/f9ec0f8cc6a137aaa05b746ca63724c12e040198/components/dashboard/src/settings/Account.tsx#L50)].

See [relevant discussion](https://gitpod.slack.com/archives/C02CBKZEVS8/p1638795733242600?thread_ts=1638366112.229500&cid=C02CBKZEVS8) (internal) and https://github.com/gitpod-io/gitpod/pull/7031#discussion_r766124754. Cc @gitpod-io/engineering-ide 

## Screenshots

Some preferences options have been removed to demonstrate this better.

| BEFORE | AFTER |
|-|-|
| Screenshot 2022-01-26 at 5 08 59 PM | Screenshot 2022-01-26 at 5 09 36 PM |

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **gtsiolis** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by JanKoehnlein — APPROVED** [review]
/lgtm

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 54a96882cc033dfcfee4c16da2cf10e517a4e1b4</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7842#pullrequestreview-864494964" title="LGTM">JanKoehnlein</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7842#issuecomment-1022944778" title="Approved">JanKoehnlein</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7842#issuecomment-1022944778" title="Approved">JanKoehnlein</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [JanKoehnlein]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Danke, @JanKoehnlein! ⚾

**Comment by JanKoehnlein** [comment]
Παρακαλώ, @gtsiolis

**Pull request merged**
